app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express")
+const app = express();
+
+const { getTopics } = require('./controllers/app.controllers')
+
+app.get('/api/topics', getTopics)
+
+app.all('/*', (req, res, next) => {
+    res.status(404).send({ msg: 'Path not found' })
+  })
+
+app.use((err, req, res, next) => {
+    if(err.status == '500') {
```

Comment:
Please remove this if statement, as we should be able to catch any kind of errors and a 'status' key is not guaranteed on all kinds of errors. Instead you could log the error in this line before sending the response.